src/lib/keychain/browser.js

```diff
@@ -1,30 +0,0 @@
-/* global window */
-// @flow
-
-/**
- * Internal dependencies
- */
-import type { Keychain } from './keychain';
-
-export default class Secure implements Keychain {
```

Comment:
There is no sense in supporting the browser keychain because we cannot run in a browser.